### PR TITLE
Lens tests

### PR DESCRIPTION
**Pull request by nico-olivares**
-Added one missing test and organized the rest.
-Please pay attention to a group of tests that are under the domLens object. I named those `domLens.hover`, `domLens.focus`, etc. I'm not sure about that so please comment.

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Branch being merged does not follow [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>You reduced the total lines of code! Awesome! :+1:</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Your PR has no browser errors. Great job!</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c998aeb6b6995add4f818837a013705e3998bf9c
</p>

File: test/lens.spec.js [hunk 1]
```diff
@@ -42,24 +42,21 @@ describe('Lens Functions', () => {
...
-      let l = F.lensOf({
-        a: 1,
-      })
-      expect(l.a.get()).to.equal(1)
-      l.a.set(5)
-      expect(l.a.get()).to.equal(5)
+      let object = { a: 1 }
+      let lens = F.lensOf(object)
+      lens.a.set(8)
+      expect(lens.a.get()).to.deep.equal(8)
```
Comment (stellarhoof) [inline test/lens.spec.js:59]: Deep equal is for objects with more than one level of nesting. It's not necessary for primitives.

Comment (nico-olivares) [reply test/lens.spec.js]: Thank you @stellarhoof. I fixed it.

**Comment by nico-olivares** [comment]
@daedalus28 The branch is up to date and the tests are valid. The PR is ready for revision.

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,6 +44,7 @@
 - Library issue resolution, two libraries where auto-updating minor versions and no longer compatible in those versions.
   - Added ignore for yarn as NPM is being used currently
   - Added versioning of package-lock files to resolve dependency issue.
+    > > > > > > > master
```
Comment (stellarhoof) [inline CHANGELOG.md:47]: Leftover

File: test/lens.spec.js [hunk 3]
```diff
@@ -242,25 +205,24 @@ describe("Lens Functions", () => {
     })
   })
   describe("domLens", () => {
-    it("value", () => {
+    it("domLens.value", () => {
```
Comment (stellarhoof) [inline test/lens.spec.js:208]: No need for `domLens.` since the group is already named `domLens`

File: CHANGELOG.md (hunk 2 above)
Comment (nico-olivares) [reply CHANGELOG.md:47]: Good find, thank you.

File: test/lens.spec.js (hunk 3 above)
Comment (nico-olivares) [reply test/lens.spec.js:208]: @stellarhoof The it(string, ) has to match the title that appears in the docs. That's how they get matched. In the docs we have F.domLens.value and more, so the `it` has to be it('domLens.value', ...). Even so, sometimes the test is not found and hence there's no example for a function.

Comment (stellarhoof) [reply test/lens.spec.js:208]: Got it

**Review by stellarhoof — APPROVED** [review]

**Pull request merged**